internal/controller/common/utils.go

```diff
@@ -186,6 +186,34 @@ func ConditionsUpdate(r client.Client, ctx context.Context, req ctrl.Request, lo
 	return nil
 }
 
+func CheckRunningPodLabels(r client.Client, ctx context.Context, namespace string, matchingLabels client.MatchingLabels) (bool, error) {
+	podList := &corev1.PodList{}
+	falconSelector := client.MatchingLabels{
+		"crowdstrike.com/provider": "crowdstrike",
```

Comment:
Is our aim to have a `namespace` that does not have any falcon resources only or not have any other non-falcon resources also in that same namespace?
* IIRC, Operator does a delete of namespace on removal. Would it not impact other non-falcon resources deployed in the same namespace? Or is this incorrect?

Reply:
Right. I will update so that there are only falcon resources and only the correct falcon resources are deployed to that namespace.

Reply:
Done